run_galaxy_workflow_tests.sh

```diff
@@ -1,9 +1,22 @@
 #!/bin/bash
 : ${PLANEMO_OPTIONS:=""}  # e.g. PLANEMO_OPTIONS="--verbose"
 
+# Ensure we can find this history later.
+nonce=$(date +%s)
+history_name="$1 $nonce"
+
 # Run test.
-planemo  $PLANEMO_OPTIONS test \
+set +e # Do not die if planemo returns non-zero
+planemo $PLANEMO_OPTIONS test \
+	--history_name "$history_name" \
 	--galaxy_url "https://usegalaxy.eu" \
 	--galaxy_user_key "$GALAXY_USER_KEY" --no_shed_install \
 	--engine external_galaxy \
-	"$1"
+	"$1";
+planemo_exit_code=$?
+set -e
+
+history_id=$(parsec histories get_histories --name 'CWL Target History' | jq -r .[0].id)
```

Comment:
If you use planemo-master, why is this still needed? The history name should be $history_name, isn't it?

Reply:
Urgh, that should have been `$history_name`

Reply:
Fixed in https://github.com/usegalaxy-eu/workflow-testing/commit/1555b2bee63e76e0986a67b09bc2ed7f1543cca0